firmware/application/hw/encoder.cpp

```diff
@@ -50,5 +102,6 @@ int_fast8_t Encoder::update(
     state <<= 1;
     state |= phase_1;
 
-    return transition_map[state & 0xf];
+    // dial sensitivity setting is stored in pmem
+    return transition_map[portapack::persistent_memory::config_encoder_dial_sensitivity()][state & 0xf];
```

Comment:
Mea culpa: I put a not in here to remove the bounds check and then last night I had a thought that the pmem might be invalid if the backup (RTC) battery dies.